assignment/tool/src/webapp/js/studentViewSubmission.js

```diff
@@ -31,3 +35,179 @@ ASN_SVS.undoCancel = function()
+
+    params.new_ts_record_day = new_ts_record_day;
+    params.new_ts_record_month = new_ts_record_month;
+    params.new_ts_record_year = new_ts_record_year;
+    params.new_ts_record_hour = new_ts_record_hour;
+    params.new_ts_record_minute = new_ts_record_minute;
+
+    params.regComment = tsComment;
+    params.regTime = tsTime;
+    button.classList.add("spinButton");
+    button.disabled = true;
+
+    ASN_TS_API._POST(endpoint, params, onSuccess, onError);
+};
+
+//SAK-43155
+ASN_TS_API.removeTimeSheet = function(button, onSuccess, onError)
+{
+    var endpoint = "/direct/assignment/removeTimeSheet.json";
+    var params = {};
+    var ts = document.getElementsByName("selectedTimesheet");
+    var tsassignmentId = document.getElementById("assignmentId").value;
+
+    params.selectedTimeSheets = Array.apply(null, ts).filter((el) => el.checked).map((el) => el.value);
+    params.assignmentId = tsassignmentId;
+
+    button.classList.add("spinButton");
+    button.disabled = true;
+
+    ASN_TS_API._POST(endpoint, params, onSuccess, onError);
+};
+
+//SAK-43155
+ASN_TS_API._GET = function(url, data, onSuccess, onError, onComplete)
+{
+    $.ajax(
+    {
+        type: "GET",
+        url: url,
+        data: data,
+        cache: false,
+        success: onSuccess || $.noop,
+        error: onError || $.noop,
+        complete: onComplete || $.noop
+    });
+};
+
+//SAK-43155
+ASN_TS_API._POST = function(url, data, onSuccess, onError, onComplete)
+{
+    $.ajax(
+    {
+        type: "POST",
+        url: url,
+        data: data,
+        success: onSuccess || $.noop,
+        error: onError || $.noop,
+        complete: onComplete || $.noop
+    });
+};
+
+//SAK-43155
+ASN.switchTimesheetTab = function( source )
+{
+    if (['tabAssignment', 'tabTimeSheet'].includes(source)) {
+        document.getElementById('tabAssignmentCurrent').classList.toggle('hidden');
+        document.getElementById('tabAssignmentRef').classList.toggle('hidden');
+        document.getElementById('tabTimeSheetRef').classList.toggle('hidden');
+        document.getElementById('tabTimeSheetCurrent').classList.toggle('hidden');
+        document.getElementById('StudentAssignmentCurrent').classList.toggle('hidden');
+        document.getElementById('StudentTimesheetCurrent').classList.toggle('hidden');
+    }
+};
+
+//SAK-43155
+ASN.handleAjaxAddSuccess = function(data)
+{
+    if (data.error && data.error.message) {
+        var button = document.getElementById("btnTimesheetAdd");
+        button.classList.remove("spinButton");
+        button.disabled = false;
+        var alertTsheetAddRecord = document.getElementById("alertTsheetAddRecord");
+        alertTsheetAddRecord.classList.toggle('hidden');
+        alertTsheetAddRecord.innerHTML= window.i18nProgresa[data.error.message];		
+    } else {
+        ASN.submitForm( 'addSubmissionForm', 'view', null, null );
+    }
+};
+
+//SAK-43155
+ASN.handleAjaxRemoveSuccess = function(data)
+{
```

Comment:
```suggestion
ASN.handleAjaxRemoveSuccess = function (data) {
```

Reply:
I've updated eclipse's .java and .js formatting settings to avoid this from happening anymore.